snuba/reader.py

```diff
@@ -4,32 +4,28 @@
 import re
 from abc import ABC, abstractmethod
 from typing import (
-    TYPE_CHECKING,
     Any,
     Callable,
     Generic,
     Iterator,
     Mapping,
     MutableMapping,
+    MutableSequence,
     Optional,
     Pattern,
     Sequence,
     Tuple,
+    TypedDict,
     TypeVar,
 )
 
-if TYPE_CHECKING:
```

Comment:
This was there in mid 2019 because we had not yet decided to have mypy as a hard dependency. Now it is not the case anymore.